velero-csi-plugin/util_test.go

```diff
@@ -0,0 +1,851 @@
+package main
+
+import (
+	"testing"
+
+	snapshotv1beta1api "github.com/kubernetes-csi/external-snapshotter/v2/pkg/apis/volumesnapshot/v1beta1"
+	snapshotFake "github.com/kubernetes-csi/external-snapshotter/v2/pkg/client/clientset/versioned/fake"
+	"github.com/sirupsen/logrus"
+	corev1api "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes/fake"
+)
+
+var (
+	csiStorageClass = "csi-hostpath-sc"
+	volumeMode      = "Filesystem"
+)
+
+func TestGetPVForPVC(t *testing.T) {
+	boundPVC := &corev1api.PersistentVolumeClaim{
```

Comment:
Not going to hold up this PR for it, especially since I don't think any of us have mentioned it and because it's in a separate repo but we have a library of "builders" in the Velero codebase that we've been trying to standardize on.

They're at https://github.com/vmware-tanzu/velero/tree/master/pkg/builder